exercicios/para-casa/entregas/resolução-ElisCoimbra/Personagem.js

```diff
@@ -0,0 +1,63 @@
+import { SuperPoder } from "./SuperPoder.js";
+
+class Personagem{
+    #nome;
+    #nomeDaVidaReal;
+    #poderes;
+    constructor(nome,nomeDaVidaReal){
+        this.#nome = nome;
+        this.#nomeDaVidaReal = nomeDaVidaReal;
+        this.#poderes = []
+    }
+    get nome(){
+        return this.#nome;
+    }
+    get nomeDaVidaReal(){
+        return this.#nomeDaVidaReal;
+    }
+   
```

Comment:
em adicionarPoder, eu deixaria mais clean, usando um operador logico. tipo: adicionarSuperPoder(superPoder) {
  if (superPoder instanceof SuperPoder && this.#poderes.length < 4 && !this.#poderes.includes(superPoder)) {
    this.#poderes.push(superPoder);
  }
}